Client/DeviceCheck/DeviceCheck.swift

```diff
@@ -344,58 +310,64 @@ private extension DeviceCheckClient {
       case .setAttestation(let nonce, _):
         let nonce = nonce.addingPercentEncoding(withAllowedCharacters: .urlPathAllowed) ?? nonce
         return URL(string: "/v1/devicecheck/attestations/\(nonce)", relativeTo: client.baseURL)
+      case .solveCaptcha(let paymentId, let captchaId, _):
+        return URL(string: "/v3/captcha/solution/\(paymentId)/\(captchaId)", relativeTo: client.baseURL)
```

Comment:
I wonder if we need to add any safeguards or sanitization here like above with `nonce` percent encoding?

Reply:
I can add some protections for the path

Reply:
Added similar protections as nonce